Lib/ctypes/util.py

```diff
@@ -341,6 +448,61 @@ def find_library(name):
             return _findSoname_ldconfig(name) or \
                    _get_soname(_findLib_gcc(name)) or _get_soname(_findLib_ld(name))
 
+
+# other systems use functions common to Linux and a few other Unix-like systems
+# https://man7.org/linux/man-pages/man3/dl_iterate_phdr.3.html
+# https://man.freebsd.org/cgi/man.cgi?query=dl_iterate_phdr
+# https://man.openbsd.org/dl_iterate_phdr
+# https://docs.oracle.com/cd/E88353_01/html/E37843/dl-iterate-phdr-3c.html
+# this relies on find_library, which is why it is defined at the end
+if os.name == "posix" and not sys.platform.startswith("aix") and sys.platform not in {"darwin", "ios", "tvos", "watchos"}:
```

Comment:
Could we split up this line a little bit? This is too long for PEP 8
```suggestion
if os.name == "posix" and not sys.platform.startswith("aix") and sys.platform not in {"darwin", "ios", "tvos", "watchos"}:
```